src/components/assetsManager/AddTokenDialog/AddTokenForm.jsx

```diff
@@ -208,7 +208,7 @@ export default class AddTokenForm extends React.Component {
     const amount = this.props.formValues && this.props.formValues.get('amount')
     const description = this.props.formValues && this.props.formValues.get('description')
     const platform = this.props.formValues && this.props.formValues.get('platform')
-    const renderPlatform = (platform) => {
+    const renderPlatform = platform => {
```

Comment:
New config for rules should allow parens for arrow functions. Check yours local cache for package `eslint-plugin-chronobank-react` branch MINT-588